src/main/java/com/nha/abdm/wrapper/hip/hrp/hipLink/HipLinkInterface.java

```diff
@@ -0,0 +1,19 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.hip.hrp.hipLink;
```

Comment:
instead of hipLink lets call this package as hipinitiated under the package link and anything wrt userinitiated, we create a package named userinitiated and place there.

Reply:
Added two packages userInitiated and hipInitiated under link